### PR TITLE
Add polynice 0.7

### PR DESCRIPTION
**Pull request by jooray**

### CODE REVIEW

File: Library/Formula/polynice.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Polynice < Formula
+  homepage "https://github.com/garabik/polynice"
+  url "https://github.com/garabik/polynice/archive/v0.7.tar.gz"
+  sha256 "7544a74203f77dc2b4a514052cf89fb5c9939e8fee0f06f52816b6a140defc07"
+
+  depends_on "python3"
```
Comment (DomT4) [inline Library/Formula/polynice.rb]: ` :python3 `, I think. @tdsmith?

Comment (tdsmith) [inline Library/Formula/polynice.rb]: Yes, please.

File: Library/Formula/polynice.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  def install
+    inreplace "polynice", "#!/usr", "#!#{HOMEBREW_PREFIX}"
```
Comment (tdsmith) [inline Library/Formula/polynice.rb]: Can we use `/usr/bin/env python3`?

**Comment by tdsmith** [comment]
Unless another maintainer wants to advocate for this: polynice is a month old, has 12 commits, and I can't find it on Google; packaging it with Homebrew [might be premature](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#niche-or-self-submitted-stuff). Setting up [a tap](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/brew-tap.md) or making it installable with pip would be great ways to distribute updates for polynice until it gains more traction.

**Comment by mistydemeo** [comment]
@tdsmith That makes sense to me - I'm not sure this (yet) meets our usual notability criteria.

**Comment by mikemcquaid** [comment]
Passing on this for now, sorry!

**Pull request closed without merging**